api/api/utils/watermark.py

```diff
@@ -190,8 +192,9 @@ def _print_attribution_on_image(img: Image.Image, image_info):
 
     font = ImageFont.truetype(_get_font_path(), size=font_size)
 
+    lic_slug, _ = get_license_slug_enum(image_info["license"])
```

Comment:
Should this also handle the case when `None` is returned?

Reply:
If we return the image as is, this would be confusing to the user. Maybe return an error? 
Admittedly, this endpoint is deprecated, and we probably never have such a case. But if we do, with the current approach, we will return 500.